### PR TITLE
{data} [intel/2016b] Keras/1.0.8 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Needs #3579

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2625.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dfcba835bc17ef542deccea3bb627b87 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/660309191664715fa480fcce7402e574 for a full test report.

File: easybuild/easyconfigs/p/PyYAML/PyYAML-3.12-intel-2016b-Python-3.5.2.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+homepage = "https://pypi.python.org/pypi/PyYAML/"
+description = """PyYAML is a YAML parser and emitter for the Python programming language."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+versionsuffix = '-Python-%(pyver)s'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyYAML/PyYAML-3.12-intel-2016b-Python-3.5.2.eb]: @wpoely86 please move this up right below `version`

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2021.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/79c42b9254902a69cf853013b91ffcde for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e4283f08f943eb22d42ed738e37b9d6c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**